flink-core/src/main/java/org/apache/flink/configuration/description/HtmlFormatter.java

```diff
@@ -34,9 +36,22 @@ protected void formatLineBreak(StringBuilder state) {
 	}
 
 	@Override
-	protected void formatText(StringBuilder state, String format, String[] elements) {
+	protected void formatText(
+			StringBuilder state,
+			String format,
+			String[] elements,
+			EnumSet<TextElement.TextStyle> styles) {
 		String escapedFormat = escapeCharacters(format);
+
+		String prefix = "";
+		String suffix = "";
+		if (styles.contains(TextElement.TextStyle.CODE)) {
+			prefix = "<tt>";
```

Comment:
`<tt>` is not part of the HTML5 standard: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tt